features/mbedtls/platform/COMPONENT_PSA_SRV_IMPL/inc/default_random_seed.h

```diff
@@ -0,0 +1,45 @@
+
+
+#ifndef MBED_OS_RTOS_COMPONENT_PSA_SRV_IMPL_DEFAULTRANDOMSEED_H_
+#define MBED_OS_RTOS_COMPONENT_PSA_SRV_IMPL_DEFAULTRANDOMSEED_H_
```

Comment:
This define name is really weird and doesn't match here the file is. Please revise. Perhaps "DEFAULT_RANDOM_SEED_H" would be fine. Look at other files for an example.